src/qr.rs

```diff
@@ -37,6 +37,10 @@ impl Into<Lot> for Error {
     }
 }
 
+pub fn starts_with_ignore_case(string: &str, pattern: &str) -> bool {
+    string.starts_with(&pattern.to_uppercase()) || string.starts_with(&pattern.to_lowercase())
```

Comment:
ignoring case usually also means accepting mixed case, eg `OpenPgp4Fpr` in this case. so maybe better do a `string.to_lowercase.starts_with(&pattern.to_lowercase())